websites/S/SOAP2DAY/dist/metadata.json

```diff
@@ -0,0 +1,24 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "SOAP2DAY",
+  "author": {
+    "name": "fun",
+    "id": "217348698294714370"
+  },
+  "category": "videos",
+  "logo": "https://i.imgur.com/WAOvAe6.png",
+  "thumbnail": "https://i.imgur.com/AEHBaRS.png",
+  "version": "1.0.0",
+  "color": "#9eeae7",
+  "tags": [
+    "stream",
+    "soap2day",
+    "movies",
+    "tv",
+    "friends"
+  ],
+  "description": {
+    "en": "You can watch all videos without logging in or registration. Soap2day is free all the time, but there are few ads on videoplayer page."
+  },
+  "regExp": "(soap2day|s2dfree)[.]"
```

Comment:
Regex is still wrong, I suggest you look into how regex work, reference other presences, and test it [here](https://regex101.com/)

Reply:
You also still need at least 1 URL, as fallback for if for some reason the regex fails.